sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -320,6 +322,27 @@ def test_results_filenames():
         assert os.path.isfile(hmiurl)
 
 
+@pytest.mark.remote_data
+def test_max_parallel_connections():
+    responses = client.search(
+        a.Time('2014/1/1T1:00:36', '2014/1/1T01:01:38'),
+        a.jsoc.Series('hmi.M_45s'), a.jsoc.Notify('jsoc@cadair.com'))
+
+    path = tempfile.mkdtemp()
+
+    with mock.patch(
+                    "parfive.Downloader.download",
+                    new_callable=mock.MagicMock
+                  ) as download:
+
+        download.side_effect = ["Mocked Downloader"]
+
+        with pytest.warns(SunpyUserWarning):
+            client.fetch(responses, path=path, max_conn=5, max_splits=5)
```

Comment:
We need to think about this a little more, as the images are still being staged for export here which is something we should try and avoid.

Reply:
I guess we could use `as-is` as the export method and then it wouldn't stage them, that might be the easiest way.

Reply:
Sorry, it took three days. I was kinda out of action.

Reply:
No worries!